transit_odp/pipelines/tests/test_dataset_etl/test_etl_vehicle_journey.py

```diff
@@ -138,50 +138,52 @@ def test_extract(self):
             list(vehicle_journey_expected.columns),
         )
 
-    def test_transform(self):
-        # setup
-        extracted = self.xml_file_parser._extract(self.doc, self.file_obj)
-        file_id = self.xml_file_parser.file_id
-
-        # test
-        transformed = self.feed_parser.transform(extracted)
-
-        vehicle_journey_expected = pd.DataFrame(
-            [
-                {
-                    "file_id": file_id,
-                    "departure_time": None,
-                    "journey_pattern_ref": "UZ000CALC:53M-jp_1",
-                    "line_ref": "CALC:UZ000CALC:53M:53M",
-                    "journey_code": None,
-                    "vehicle_journey_code": "vj_1",
-                    "service_code": "UZ000CALC:53M",
-                    "direction": "inbound",
-                },
-            ]
-        ).set_index("file_id")
-        self.assertTrue(
-            check_frame_equal(transformed.vehicle_journeys, vehicle_journey_expected)
-        )
-
-        self.assertCountEqual(
-            list(transformed.vehicle_journeys.columns),
-            list(vehicle_journey_expected.columns),
-        )
-
-    def test_load(self):
-        extracted = self.xml_file_parser._extract(self.doc, self.file_obj)
-        transformed = self.feed_parser.transform(extracted)
-
-        # test
-        self.feed_parser.load(transformed)
-
-        vehicle_journeys = VehicleJourney.objects.all()
-
-        self.assertEqual(1, vehicle_journeys.count())
-        for journey in vehicle_journeys:
-            self.assertEqual(journey.line_ref, "CALC:UZ000CALC:53M:53M")
-            self.assertEqual(journey.journey_code, None)
+    # TODO: need to fix the below test case as part of DQS ticket
```

Comment:
Can we used `@pytest.mark.skip` instead of commented code?

Reply:
done